sdk/search/azure-search/src/main/java/com/azure/search/implementation/SearchIndexRestClientBuilder.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
```

Comment:
We don't append "Changes may case.... regenerated" line in auto rest header across other Track 2 SDKs.
@JonathanGiles , @srnagar 
Is there a strict requirement for header to be limited to just the first three lines ?

Reply:
the header is injected during the code generation. if the text need to be revised, it will also trigger a change in autorest config for search https://github.com/Azure/azure-rest-api-specs/tree/master/specification/search/data-plane. Please advice

Reply:
the header is injected during the code generation. if the text need to be revised, it will also trigger a change in autorest config for search https://github.com/Azure/azure-rest-api-specs/tree/master/specification/search/data-plane. Please advice